Assets/Scripts/Factories/PlantFactory.cs

```diff
@@ -6,13 +6,14 @@
 
 public class PlantFactory : AbstractGameObjectFactory
 {
-    public override void CreateGameObject(Vector3 thePosition, float scale)
+    public override GameObject CreateGameObject(Vector3 thePosition, float scale)
```

Comment:
Nice catch, this will be useful when we need to add to the singleton for the plant objects.